diff --git a/node_modules/jest-snapshot/build/index.js b/node_modules/jest-snapshot/build/index.js
index 9094081..db825d4 100644
--- a/node_modules/jest-snapshot/build/index.js
+++ b/node_modules/jest-snapshot/build/index.js
@@ -504,7 +504,10 @@ const toThrowErrorMatchingInlineSnapshot = function (
   return _toThrowErrorMatchingSnapshot(
     {
       context: this,
-      inlineSnapshot,
+      inlineSnapshot:
+        inlineSnapshot !== undefined
+          ? stripAddedIndentation(inlineSnapshot)
+          : undefined,
       isInline: true,
       matcherName,
       received
